chapter10/axp/ex02

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+use Benchmark qw(timethese);
+
+chdir( $ARGV[0] // "~" );
+
+timethese( 5, { naive => \&naive, schwartzian => \&schwartzian } );
```

Comment:
When I cheated and used your solutions as a sanity check (I attributed your work in a comment of mine so it's *totally* fine), it complained to me that 5 wasn't enough iterations for it to really report anything